yt_dlp/extractor/facebook.py

```diff
@@ -555,8 +585,8 @@ def extract_relay_data(_filter):
         def extract_relay_prefetched_data(_filter):
             return traverse_obj(extract_relay_data(_filter), (
                 'require', (None, (..., ..., ..., '__bbox', 'require')),
-                lambda _, v: 'RelayPrefetchedStreamCache' in v, ..., ...,
-                '__bbox', 'result', 'data', {dict}), get_all=False) or {}
+                lambda _, v: any(key.startswith('RelayPrefetchedStreamCache') for key in v if isinstance(key, str)),
```

Comment:
this should be enough
```suggestion
                lambda _, v: any(key.startswith('RelayPrefetchedStreamCache') for key in v),
```

Reply:
IDE doesn't provide good type detection without it, but okay.